src/frontend/components/UI/DialogHandler/components/MessageBoxModal/index.tsx

```diff
@@ -72,7 +75,15 @@ const MessageBoxModal: React.FC<MessageBoxModalProps> = function (props) {
       <DialogHeader onClose={props.onClose}>{props.title}</DialogHeader>
       <DialogContent className="body dialogContent">
         {getContent()}
+        {props.showCheckbox && (
```

Comment:
This will show a warning for games launched using compatibility layer.
It includes a 'Do not show again' button. This is stored individually for each game so the person will see this warning when trying a new windows game on mac or linux.